node/src/main/scala/coop/rchain/node/web/WebApiRoutes.scala

```diff
@@ -36,7 +36,12 @@ object WebApiRoutes {
             case err: BlockApiException => BadRequest(err.getMessageSafe.asJson)
             case err: Throwable         =>
               // Logging only unanticipated errors, not related to Block API or input parsing (user errors)
-              Log[F].error("HTTP API response error", err) *> BadRequest(err.getMessageSafe.asJson)
+
+              // To indicate that error is not related to internal node error and node can continue running,
+              // the stack is cleared and error logged as a warning.
+              Sync[F].delay(err.setStackTrace(Array.empty[StackTraceElement])) *>
```

Comment:
Why is the reason to clear stack trace?

Reply:
This is related to your [comment](https://github.com/rchain/rchain/issues/3740#issuecomment-1158930726). It is looks like usual log message, not a fatal error.
But now I'm thinking it is output just for node admin, not for client, so it is better to provide maximum information about error.

Reply:
Yes, exactly. Especially because this is unexpected API error which is not fatal, but node operator should pay some attention and report upstream.